tests/pdctl/scheduler/scheduler_test.go

```diff
@@ -481,14 +481,4 @@ func TestScheduler(t *testing.T) {
 	err = leaderServer.GetServer().SetScheduleConfig(*cfg)
 	re.NoError(err)
 	checkSchedulerWithStatusCommand(nil, "disabled", nil)
-
-	// test split bucket scheduler
-	echo = mustExec([]string{"-u", pdAddr, "scheduler", "config", "split-bucket-scheduler"}, nil)
-	re.Contains(echo, "\"degree\": 3")
-	echo = mustExec([]string{"-u", pdAddr, "scheduler", "config", "split-bucket-scheduler", "set", "degree", "10"}, nil)
-	re.Contains(echo, "Success")
-	echo = mustExec([]string{"-u", pdAddr, "scheduler", "config", "split-bucket-scheduler"}, nil)
-	re.Contains(echo, "\"degree\": 10")
-	echo = mustExec([]string{"-u", pdAddr, "scheduler", "remove", "split-bucket-scheduler"}, nil)
-	re.Contains(echo, "Success!")
```

Comment:
In current, the `bucket-split-scheduler` scheduler maybe wrong, it may bring too many split region even if the hot loads don't need to be scheduled.
So in this pr, the buckets split logical function have integrate together with the hot region scheduler. only when the hot region scheduler find the moved region is too big,  the bucket split can work on the moved region. 
It doesn't brings superfluous split if the hot loads don't need to be scheduled.